udf/pos_of_trailing_set_bit.sql

```diff
@@ -1,5 +1,7 @@
 /*
 
+Identical to bits28_days_since_seen.
```

Comment:
Ah, it is noted! Still seems odd to have two identical functions. Maybe in the next few weeks we can deprecate this one.